data/translations/en/register.json

```diff
@@ -26,6 +26,7 @@
   "dates__bymail_deadline": "Register by mail deadline:",
   "dates__byonline_deadline": "Online registration deadline:",
   "dates__electionday": "General Election Day: Tuesday, November 3, 2020.",
+  "dates__electionday_la": "General Election Day: Saturday, December 10, 2022.",
```

Comment:
Can we add a dates__electionday property to the state-data json for louisiana? That way any state can have an override for election day.